src/ussd_popi_rapidpro.js

```diff
@@ -993,76 +1152,33 @@ go.app = function() {
-                    new Choice("state_delete_data", $("No"))
-                ]
-            });
-        });
-
-        self.add("state_loss_delete_data", function(name) {
-            return new ChoiceState(name, {
-                question: $(
-                    "You'll get support msgs. We hold your info for " +
-                    "historical/research/statistical reasons. Do you want us to delete it after " +
-                    "you stop getting msgs?"
-                ),
-                error: $(
-                    "Sorry we don't recognise that reply. Please enter the number next to your " +
-                    "answer."
-                ),
-                choices: [
-                    new Choice("yes", $("Yes")),
-                    new Choice("no", $("No"))
-                ],
-                next: "state_submit_opt_out"
-            });
-        });
-
-        self.add("state_delete_data", function(name) {
-            return new MenuState(name, {
-                question: $(
-                    "We hold your info for historical/research/statistical reasons after you " +
-                    "opt out. Do you want to delete your info after you stop getting messages?"
-                ),
-                error: $(
-                    "Sorry we don't recognise that reply. Please enter the number next to your " +
-                    "answer."
-                ),
-                choices: [
-                    new Choice("state_delete_confirm", $("Yes")),
+                    new Choice("state_submit_opt_out", $("Yes")),
                     new Choice("state_submit_opt_out", $("No"))
                 ]
             });
         });
 
-        self.add("state_delete_confirm", function(name) {
-            return new MenuState(name, {
-                question: $(
-                    "All your info will be permanently deleted in the next 7 days. We'll stop " +
-                    "sending you messages. Please select Next to continue:"
-                ),
-                error: $(
-                    "Sorry we don't recognise that reply. Please enter the number next to your " +
-                    "answer."
-                ),
-                choices: [
-                    new Choice("state_submit_opt_out", $("Next"))
-                ]
+        self.add("state_message_unhelpful_or_unknown", function(name) {
+            return new EndState(name, {
+                next: "state_start",
+                text: $("Thank you for supporting MomConnect. You won't get any more messages from us." +
+                        "\n" +
+                        "For any medical concerns, please visit a clinic."
+                )
             });
         });
 
         self.add("state_submit_opt_out", function(name, opts) {
             var msisdn = utils.normalize_msisdn(self.im.user.addr, "ZA");
             var answers = self.im.user.answers;
-            var forget = answers.state_delete_data === "state_delete_confirm";
```

Comment:
I think we can leave this in since the forget should also submit through here.

Reply:
Ideally we shouldn't change this payload at all, this functionality isn't changing, just the way we present the questions